docs/docs.md

```diff
@@ -13,11 +13,14 @@ from dynamics import DynamicsClient
 - client_id: str - Client id (e.g. UUID).
 - client_secret: str - Client secret (e.g. OAuth password).
 - scope: List[str] - List of urls in form: 'https://[Organization URI]/scope'. Defines the database records that the API connection has access to.
+- *resource: Optional[str] - Url in form: 'https://[Organization URI]/scope'. Defines the database records that the API connection has access to.*
```

Comment:
Shouldn't URL be in form `https://[Organization URI]/`? At least that's what the [client docstring](https://github.com/MrThearMan/dynamics-client/pull/4/files#diff-fafb0b7303b9394168b2c6f6aef7be16bcc89b26745d80f08eaa4f03eb37bb5bR110) says.

Reply:
thank you